lib/dom_html.ml

```diff
@@ -0,0 +1,727 @@
+
+  let onDragEnter = (event "onDragEnter" : (Event.Mouse.t -> unit) -> t)
+
+  let onDragExit = (event "onDragExit" : (Event.Mouse.t -> unit) -> t)
+
+  let onDragLeave = (event "onDragLeave" : (Event.Mouse.t -> unit) -> t)
+
+  let onDragOver = (event "onDragOver" : (Event.Mouse.t -> unit) -> t)
+
+  let onDragStart = (event "onDragStart" : (Event.Mouse.t -> unit) -> t)
+
+  let onDrop = (event "onDrop" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseDown = (event "onMouseDown" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseEnter = (event "onMouseEnter" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseLeave = (event "onMouseLeave" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseMove = (event "onMouseMove" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseOut = (event "onMouseOut" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseOver = (event "onMouseOver" : (Event.Mouse.t -> unit) -> t)
+
+  let onMouseUp = (event "onMouseUp" : (Event.Mouse.t -> unit) -> t)
+
+  (* Selection events *)
+
+  let onSelect = (event "onSelect" : (Event.Selection.t -> unit) -> t)
+
+  (* Touch events *)
+
+  let onTouchCancel = (event "onTouchCancel" : (Event.Touch.t -> unit) -> t)
+
+  let onTouchEnd = (event "onTouchEnd" : (Event.Touch.t -> unit) -> t)
+
+  let onTouchMove = (event "onTouchMove" : (Event.Touch.t -> unit) -> t)
+
+  let onTouchStart = (event "onTouchStart" : (Event.Touch.t -> unit) -> t)
+
+  (* Pointer events *)
+
+  let onPointerOver = (event "onPointerOver" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerEnter = (event "onPointerEnter" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerDown = (event "onPointerDown" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerMove = (event "onPointerMove" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerUp = (event "onPointerUp" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerCancel =
+    (event "onPointerCancel" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerOut = (event "onPointerOut" : (Event.Pointer.t -> unit) -> t)
+
+  let onPointerLeave = (event "onPointerLeave" : (Event.Pointer.t -> unit) -> t)
+
+  let onGotPointerCapture =
+    (event "onGotPointerCapture" : (Event.Pointer.t -> unit) -> t)
+
+  let onLostPointerCapture =
+    (event "onLostPointerCapture" : (Event.Pointer.t -> unit) -> t)
+
+  (* UI events *)
+
+  let onScroll = (event "onScroll" : (Event.UI.t -> unit) -> t)
+
+  (* Wheel events *)
+
+  let onWheel = (event "onWheel" : (Event.Wheel.t -> unit) -> t)
+
+  (* Media events *)
+
+  let onAbort = (event "onAbort" : (Event.Media.t -> unit) -> t)
+
+  let onCanPlay = (event "onCanPlay" : (Event.Media.t -> unit) -> t)
+
+  let onCanPlayThrough =
+    (event "onCanPlayThrough" : (Event.Media.t -> unit) -> t)
+
+  let onDurationChange =
+    (event "onDurationChange" : (Event.Media.t -> unit) -> t)
+
+  let onEmptied = (event "onEmptied" : (Event.Media.t -> unit) -> t)
+
+  let onEncrypetd = (event "onEncrypetd" : (Event.Media.t -> unit) -> t)
```

Comment:
```suggestion
  let onEncrypted = (event "onEncrypted" : (Event.Media.t -> unit) -> t)
```

Reply:
Good catch! This is actually from `rescript-react`, so I've created PR for it upstream too.